### PR TITLE
Followup to fix display bugs introduced in #18023

### PR DESCRIPTION
**Pull request by mdlinville**
Compare with https://www.cockroachlabs.com/docs/v23.1/secure-cockroachdb-kubernetes?filters=helm#deploy-cert-manager-for-mtls

### CODE REVIEW

File: src/current/v22.2/secure-cockroachdb-kubernetes.md [hunk 1]
```diff
@@ -170,14 +168,14 @@ Complete the following steps **before** [initializing the cluster]({% link {{ pa
...
-    $ kubectl get secrets
+    kubectl get secrets
     ~~~
 
     ~~~
-    NAME                      TYPE                                  DATA   AGE
-    cockroachdb.client.root   Opaque                                3      13s
-    cockroachdb.node          Opaque                                3      3s
-    default-token-6js7b       kubernetes.io/service-account-token   3      9h
+    NAME                      TYPE                                   DATA   AGE
```
Comment (kathancox) [inline src/current/v22.2/secure-cockroachdb-kubernetes.md:175]: In each of the versions, this output is misaligned so it's bit hard to read

File: src/current/v22.2/secure-cockroachdb-kubernetes.md [hunk 2]
```diff
@@ -512,18 +498,18 @@ If you previously [authenticated with `cockroach cert`](#example-authenticating-
 
     ~~~
     Certificate directory: certs
-      Usage  | Certificate File |    Key File     |  Expires   |                                                                                                                                  Notes                                                                                                                                  | Error
+        Usage  | Certificate File |  Key File   |  Expires   |                                                                  Notes                                                                  | Error
```
Comment (kathancox) [inline src/current/v22.2/secure-cockroachdb-kubernetes.md:501]: This output is misaligned, making it a bit hard to read.

**Review by kathancox — APPROVED** [review]
LGTM, a couple of non-blocking comments. Some of the output is misaligned.

File: src/current/v22.2/secure-cockroachdb-kubernetes.md (hunk 2 above)
Comment (mdlinville) [reply src/current/v22.2/secure-cockroachdb-kubernetes.md]: Removed the output, which is too wide to format well and where the Error column is empty for all rows.

**Pull request merged**